src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +546,67 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_update_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        [a_min, a_max] = [0.e-3, 100.e-3]
+        [b_min, b_max] = [0.e-3, 13.e-3]
+        dependency = simulated.Stage("stage_", "test", axes=["a", "b"],
+                                     ranges={"a": [a_min, a_max], "b": [b_min, b_max]})
+        # Map dependency onto a given rotation
+        tilt = math.radians(26)
+        stage = ConvertStage("conv", "align", {"orig": dependency},
+                             axes=["a", "b"], rotation=tilt)
+        # get the range of stage
+        x_min, x_max = stage.axes["x"].range
+        y_min, y_max = stage.axes["y"].range
+
```

Comment:
Can you check that (now) the dependency range is the one that was passed at init? Something like:
```python
self.assertEqual(dependency.axes["a"].range, (a_min, a_max))
...
```